guides/common/modules/proc_upgrading-el7-to-el8.adoc

```diff
@@ -0,0 +1,173 @@
+enabled=1
+gpgcheck=1
+
+ifdef::katello[]
+[leapp-katello]
+name=Katello 4.4
+baseurl=https://yum.theforeman.org/katello/4.4/katello/el8/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+
+[leapp-katello-candlepin]
+name=Candlepin: an open source entitlement management system.
+baseurl=https://yum.theforeman.org/katello/4.4/candlepin/el8/$basearch/
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+
+[leapp-pulpcore]
+name=pulpcore: Fetch, Upload, Organize, and Distribute Software Packages.
+baseurl=https://yum.theforeman.org/pulpcore/3.16/el8/$basearch/
+gpgkey=https://yum.theforeman.org/pulpcore/3.16/GPG-RPM-KEY-pulpcore
+enabled=1
+gpgcheck=1
+module_hotfixes=1
+endif::[]
+
+[leapp-foreman-plugins]
+name=Foreman plugins 3.2
+baseurl=https://yum.theforeman.org/plugins/3.2/el8/$basearch
+enabled=1
+gpgcheck=0
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman
+module_hotfixes=1
+
+[leapp-foreman-client]
+name=Foreman client 3.2
+baseurl=https://yum.theforeman.org/client/3.2/el8/$basearch
+enabled=1
+gpgcheck=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-foreman-client
+
+[leapp-puppet7]
+name=Puppet 7 Repository el 8 - $basearch
+baseurl=http://yum.puppetlabs.com/puppet7/el/8/$basearch
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-puppet7-release
+       file:///etc/pki/rpm-gpg/RPM-GPG-KEY-2025-04-06-puppet7-release
+enabled=1
+gpgcheck=1
+----
+
+* If you are using Puppet 6 instead of Puppet 7, replace the `7` with a `6` in the `leapp-puppet7` entry.
+
+* You need a Puppet repository for the Puppet agent that the installer is using.
+
+For more information, see link:https://docs.theforeman.org/nightly/Managing_Configurations_Puppet/index-katello.html[Configuring Hosts Using Puppet].
+
+[start=4]
+. We do not support Enterprise Linux 8 installations with EPEL 8 enabled, so remove `epel-release`:
++
+----
+# yum remove epel-release
+----
+[start=5]
+. Remove `centos-release-scl` and `centos-release-scl-rh` repositories:
++
+----
+# yum remove centos-release-scl centos-release-scl-rh
+----
+[start=6]
+. Let LEAPP analyze your system:
++
+----
+# leapp preupgrade
+----
+The first run is expected to fail but report issues.
+Continue to the next step for remediation.
+
+[start=7]
+. Fix issues found by LEAPP (check `/var/log/leapp/leapp-report.txt` for details):
++
+----
+# rmmod pata_acpi
+# echo PermitRootLogin yes | tee -a /etc/ssh/sshd_config
+# leapp answer --section remove_pam_pkcs11_module_check.confirm=True
+----
+
+The `preupgrade` might fail with a dependency resolution error such as:
+
+* "package rubygem-fx-0.5.0-2.el8.noarch requires rubygem(railties) >= 4.0.0, but none of the providers can be installed"
+* "package rubygem-railties-6.0.4.7-1.el8.noarch requires rubygem(thor) < 2.0, but none of the providers can be installed"
+
+If this happens, do the following to clean up packages that cannot automatically upgrade:
++
```

Comment:
these `+` render weirdly for me:

![image](https://user-images.githubusercontent.com/94284/172556400-982b5ee6-4686-4b00-9b75-95144796cbc3.png)

Reply:
Same. I will delete the +.